File: demo_langs/invalid/sample_bad.java
// Java - invalid code

public class User {
    private int id
    private String name  // brak średników

    public User(int id String name) {  // brak przecinka
        this.id = id
        this.name = name  // brak średnika
    }  // brak zamknięcia

class UserService {  // brak zamknięcia klasy User
    private java.util.List<User> users = new java.util.ArrayList<>()
    
    public void addUser(User user) {  // brak ciała metody
        users.add(user  // brak zamknięcia nawiasu i średnika
    }  // brak średnika
}  // brak średnika

File: demo_langs/invalid/sample_bad.php
<?php
// PHP - invalid code

class User {
    public int $id
    public string $name  // brak średników

    public function __construct(int $id, string $name) {  // brak ciała
        $this->id = $id
        $this->name = $name  // brak średnika
    }  // brak zamknięcia

class UserService {  // brak zamknięcia klasy User
    private array $users = []
    
    public function addUser(User $user  // brak zamknięcia nawiasu
        $this->users[] = $user  // brak średnika
    }  // brak średnika
}  // brak średnika

$service = new UserService()
$service->addUser(new User(1, "Alice"))  // brak średnika
